code/modules/research/techweb/all_nodes.dm

```diff
@@ -61,6 +61,15 @@
 					"stethoscope", "surgical_drapes", "syringe", "plumbing_rcd", "beaker", "large_beaker", "xlarge_beaker",
 					"dropper", "defibmountdefault", "surgical_tape", "portable_chem_mixer")
 
+/datum/techweb_node/basic_wiremod
```

Comment:
Can we rename this to "circuitry" instead of wiremod?

Reply:
I think the reverse should be considered instead, renaming the paths to e.g. `/obj/item/wiremod`, and the `iscomponent` check to `iswiremod`.

Reply:
But leaving the player facing names as Circuits and Circuit components of course